contracts/modules/STO/USDTieredSTO.sol

```diff
@@ -568,6 +544,17 @@ contract USDTieredSTO is USDTieredSTOStorage, STO {
         }
     }
 
+    function _isAccredited(address _investor) internal view returns(bool) {
+        IDataStore dataStore = IDataStore(getDataStore());
+        return _getIsAccredited(_investor, dataStore);
+    }
+
+    function _getIsAccredited(address _investor, IDataStore dataStore) internal view returns(bool) {
+        uint256 flags = dataStore.getUint256(_getKey(INVESTORFLAGS, _investor));
+        uint256 flag = flags & uint256(1); //isAccredited is flag 0 so we don't need to bit shift flags.
```

Comment:
Should we store the meaning for each bit in INVESTORFLAGS somewhere? Even if it is just in a comment in the DataStore, or a .md file somewhere in the repo.

Reply:
I already added it https://github.com/PolymathNetwork/polymath-core/blob/cdc941da0b7851f965a396b87c0dc212d5ec8296/docs/investor_flags.md :)

Reply:
@maxsam4 a minor typo fix in the .md file

Reply:
Pushed a fix for the typo I found. If there is any other typo present, please let me know.